WalletWasabi/Helpers/ByteHelpers.cs

```diff
@@ -1,6 +1,7 @@
-﻿using System.Collections.Generic;
+using System.Collections.Generic;
```

Comment:
I think this is here because there were changes in this file that were removed later. Can be removed?